modal/index.js

```diff
@@ -122,6 +122,19 @@ Modal.prototype.setVisible = function(enabled) {
   }
 
   var lightboxEl = document.querySelector('.' + this.config.className);
+
+  var enterClass = this.config.className + '--enter';
+  var exitClass = this.config.className + '--exit';
+  window.setTimeout(function() {
+    if(enabled) {
```

Comment:
Thanks updated space in front of (.

As for setTimeout, I'm using the to take it to the bottom of the execution queue otherwise, animations that depend on ak-modal--exit for instance, don't seem to trigger properly.